src/display/editor/drawers/freedraw.js

```diff
@@ -591,17 +246,60 @@ class FreeOutliner {
         );
       }
     }
-    buffer.push(`L${bottom[4]} ${bottom[5]} Z`);
+
+    this._toSVGPathStart(buffer);
 
     return buffer.join(" ");
   }
 
+  _toSVGPathTwoPoints() {
+    const [x, y, width, height] = this.#box;
+    const [lastTopX, lastTopY, lastBottomX, lastBottomY] =
+      this.#getLastCoords();
+
+    return `M${(this.#last[2] - x) / width} ${
+      (this.#last[3] - y) / height
+    } L${(this.#last[4] - x) / width} ${(this.#last[5] - y) / height} L${lastTopX} ${lastTopY} L${lastBottomX} ${lastBottomY} L${
+      (this.#last[16] - x) / width
+    } ${(this.#last[17] - y) / height} L${(this.#last[14] - x) / width} ${
+      (this.#last[15] - y) / height
+    } Z`;
+  }
+
+  _toSVGPathStart(buffer) {
+    const bottom = this.#bottom;
+    buffer.push(`L${bottom[4]} ${bottom[5]} Z`);
+  }
+
+  _toSVGPathEnd(buffer) {
```

Comment:
Can we make this method actually private instead?
```suggestion
  #toSVGPathEnd(buffer) {
```